tests/services/schemas/test_creator_contributor.py

```diff
@@ -330,26 +330,6 @@ def test_contributor_person_invalid_no_family_name_nor_given_name(
     )
 
 
-def test_contributor_invalid_no_role(vocabulary_clear):
```

Comment:
Removed the tes by mistake, will add it once the first review it's done. (sorry ...)